ludwig/encoders/image_encoders.py

```diff
@@ -223,52 +237,52 @@ def __init__(
             default_weights_regularizer=weights_regularizer,
             default_bias_regularizer=bias_regularizer,
             default_activity_regularizer=activity_regularizer,
-            # default_weights_constraint=fc_weights_constraint,
-            # default_bias_constraint=fc_bias_constraint,
             default_norm=norm,
             default_norm_params=norm_params,
             default_activation=activation,
             default_dropout=dropout,
         )
 
-    def call(self, inputs, training=None, mask=None):
+    def call(self, inputs):
```

Comment:
Same as above regarding `forward` and `output_shape`.

Reply:
Done!